code_testing.ts

```diff
@@ -2,7 +2,7 @@
 //this is for code meant to be executed to test the functionality of code being developed
```

Comment:
I dont think we can consider this testing. (Hayes is whispering: Manual Exploratory Black box unit tests... is that possible?)
But if you guys want to use it as a little sandbox for interacting with the backend :+1: 

If you guys want our help in creating tests for your backend, we would be willing to assist.